src/analysis/slice/data-flow.ts

```diff
@@ -337,6 +337,11 @@ export class DataflowAnalyzer {
         uses = undefinedRefs.filter(r => r.level == ReferenceType.USE);
         break;
       case ast.CLASS:
+        // For each class function, call getUses
+        const usesArr = statement.code
+          .flatMap(classStatement => this.getUses(classStatement, _));
```

Comment:
A potential fix is to just factor out `flatMap`. How about...

```suggestion
          .forEach(classStatement => uses.add(...this.getUses(classStatement, _).items);
```

Reply:
Also lets you remove the line below (`uses = ...`)

Reply:
Great point! Dropped the assignment to `usesArr` too.